CPAC/utils/serialization/workflow_pickle.py

```diff
@@ -0,0 +1,19 @@
+import pickle
+from typing import Any
+
+from .core import workflow_container
+
+
+def save_workflow_pickle(filename: str, workflow: Any) -> None:
```

Comment:
Why type `workflow` as `Any` instead of as `Workflow`?

Reply:
As the pickle method does not require the workflow to inherit from `Workflow` (could also be some kind of container object)
This function is for development+debug purposes only (I can remove the file completely, but it is handy for developing the JSON serialization as it contains _everything_ of the object)

Reply:
Oh, cool. I'd just add a note in the docstring here https://github.com/FCP-INDI/C-PAC/blob/b0fb9a97776b7c4f569246723a01bd167ee4cf48/CPAC/utils/serialization/workflow_pickle.py#L14 about what else might be expected

Reply:
Done!